### ISSUE TITLE
No documentation present

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ebouhoubeiny/cpacs#218]
```
Occurs in Type: 
landingGearType

Comments: 
No documentation present

daniel.boehnke@dlr.de
```

Original issue reported on code.google.com by `daniel.b...@googlemail.com` on 25 Jan 2012 at 10:56

**Comment by GoogleCodeExporter** [comment]
Original comment by `daniel.b...@googlemail.com` on 8 Feb 2012 at 9:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `daniel.b...@googlemail.com` on 8 Feb 2012 at 4:37
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Included in 1.7 from Felix
```

Original comment by `daniel.b...@googlemail.com` on 16 Feb 2012 at 3:26
- Changed state: **Done**
- Added labels: **cpacs-documentation**
